config/nf_codon_dbs.config

```diff
@@ -5,15 +5,24 @@
 */
 
 params {
-    busco_ref_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/busco_new"
-    CAT_ref_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/cat/CAT_prepare_20210107"
-    cat_diamond_db_name = 'Diamond_2.0.6/diamond'
-    cat_db_name = '2021-01-07_CAT_database'
-    cat_taxonomy_db = '2021-01-07_taxonomy'
-    checkm_ref_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/checkm2/CheckM2_database/CheckM2_database/uniref100.KO.1.dmnd" // "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/checkm"
-    eukcc_ref_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/eukccdb/eukcc2_db_ver_1.1"
-    gunc_ref_db = "/hps/nobackup/rdf/metagenomics/service-team/production/ref-dbs/genomes-pipeline/gunc_db_2.0.4.dmnd"
-    gtdbtk = "/hps/nobackup/rdf/metagenomics/service-team/production/ref-dbs/genomes-pipeline/release207_v2"
+    
+    assemblies = "/hps/nobackup/rdf/metagenomics/service-team/users/kates/eukrecover2/Vaginal/Assemblies/ERP138373/raw"
+    raw_reads  = "/hps/nobackup/rdf/metagenomics/service-team/users/kates/eukrecover2/Vaginal/Raw_reads/SRP287478/raw/"
+
+    erz_to_err_mapping_file = "/hps/nobackup/rdf/metagenomics/service-team/users/kates/eukrecover2/Vaginal/rename.tsv"
+
+    checkm2_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/checkm2/CheckM2_database/CheckM2_database/uniref100.KO.1.dmnd"
+
+    gunc_db = "/hps/nobackup/rdf/metagenomics/service-team/production/ref-dbs/genomes-pipeline/gunc_db_2.0.4.dmnd"
+    gtdbtk_db = "/hps/nobackup/rdf/metagenomics/service-team/production/ref-dbs/genomes-pipeline/release207_v2"
+
+    busco_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/busco_new"
+    eukcc_db = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/eukccdb/eukcc2_db_ver_1.1"
```

Comment:
we should switch to version 1.2 (located in the same folder)

Reply:
done